pkg/workloadmeta/collectors/kubelet/stub.go

```diff
@@ -0,0 +1,6 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package kubelet
```

Comment:
Since we import `pkg/workloadmeta/collectors/kubelet` explicitly, and the other file in the package has a `+build kubelet`, not having this file will fail with "no Go files in package" when building the agent for non-kubelet builds.